README.md

```diff
@@ -42,6 +42,9 @@ Add the following CSS snippet to `styles/app.css`:
 }
 
 .infinite-list-content {
+  box-sizing: initial;
+  -moz-box-sizing: initial;
+  -webkit-box-sizing: initial;
```

Comment:
I don't actually think this is what we want. The css that's included is 100% theirs for their app. I think it should be up to the developer to know their CSS since it's all very different. 

Using `initial` in CSS can have surprising results when dealing with inheritance.